server/model/provider.model.js

```diff
@@ -14,5 +14,11 @@ module.exports = {
             .select('*')
             .where('id', provideId)
             .timeout(1500);
+    },
+    providerIdByName(providerName) {
```

Comment:
I would say that the name of the function should be a clear name of what the function does.

examples
- getUserId
- createAccount
- deleteAccount
- createPost